iptables/iptables.go

```diff
@@ -35,6 +42,7 @@ const (
 
 var (
 	iptablesPath  string
+	iptablesVer   version
```

Comment:
Wanted to keep it rooted to the version, so that if in future we have to run some commands which are available only starting from a specific version, we only need to add a new internal method which does the check, instead of adding a new flag for each feature.

But I am not strong about this, if you think it is still better to just store the flag. Let me know.